data/json/bionics.json

```diff
@@ -650,25 +645,26 @@
     "id": "bio_lighter",
     "type": "bionic",
     "name": { "str": "Finger Lighter" },
-    "description": "The index fingers of both hands have powerful fire starters which extend from the tip.",
-    "occupied_bodyparts": [ [ "hand_l", 1 ], [ "hand_r", 1 ] ],
+    "description": "The index finger of your left hand has fire starter which extend from the tip.",
+    "occupied_bodyparts": [ [ "hand_l", 1 ] ],
     "fake_item": "fake_firestarter",
```

Comment:
This should really be noted as part of the integrated toolset.

Reply:
Rule of cool. Finger lighter is just too recognizable sci-fi cyborg gizmo

Reply:
Point.